### ISSUE TITLE
[SUGGESTION] Ability to turn off the "It seems that you are stuck!" error message. 

### ISSUE THREAD

**Issue by Servomotor** [tsunamayo/Starship-EVO#4504]
### Short Description of the issue:

May we get an F3 debug menu option to turn off the "It seems that you are stuck!" message? 

 Because of the annoying popup, I spent most of my time in no clip mode, but I have noticed this is causing issues with some of my building because I tend to clip about half way through the floor which makes the ceiling feel taller than it is while I am building. I will often turn no clip off to get a feel for how my character operates in the environment, but the second the "stuck" messsage pops up, I just go back into no clip mode. 
 
 Thank you. Hope you are well and safe with no damage or injury after the earthquake. 
![20220122114047_1](https://user-images.githubusercontent.com/65745916/150649749-fa5f61ce-9b28-4f35-9da7-92e88794996c.jpg)

**Comment by tsunamayo** [comment]
@Servomotor Hi, thanks, are you talking about the Tonga one? No problem here!
Well my issue is why are you getting stuck often! This is the bug I would like to fix. Have you a case where this happen in a reproducible way?
Thanks

**Comment by Servomotor** [comment]
Check your most recent email from me. There is a ship attached in that email that does this. Just walk around without jetpack on and it will eventually give the "stuck" error message.

**Comment by tsunamayo** [comment]
@Servomotor okay, also to confirm what kind of FPS do you have in game? thx

**Comment by Servomotor** [comment]
Good morning boss. I get a consistent 60 FPS. I run a design monitor, so I cap it at 60 (but I always get 60 in game...except on Ulty ships, lol). 

I dont detect a performance drop when the stuck error message comes up. Everything seems to be normal, I just get the error message. 

I thought it was because I am using the Panel Rounded on a lot of surfaces, but I am not sure.

**Comment by tsunamayo** [comment]
@Servomotor By the way this feature was added a long time back. Thanks I hope you are doing well!

**Comment by Servomotor** [comment]
Hey Francois! Thank you for the wishes. Hope your baby, wife and yourself are doing well.

My girlfriends mum has pancreatic cancer. The doctors waited too long to work on her/remove the cancer so she is in a hospice basically waiting to die. They can’t do kemo because she is weak. So, life has been kinda tough atm.

I have dabbled here and there in game. I have been playing with the new hoses on Captain Ahd’s ship, when the hoses go live I will update the blueprint.

I have also worked on those station modules I showed you awhile back. I need to work on the central hub and then some vertical decorative modules. The basic idea being that the player would only be traversing on a horizontal plane, no elevators or stairs needed. But on the exterior the modules would still build above and below the central hub. This way it can have a tall vertical feel, but not need elevators and stuff. I am just trying to keep it simple.

I also moved away from the silver. Its more of a grey scale now with a splash of color. I want to add some hoses to these modules as well.

The 2 way, 3 way, 4 way, 6 way, dry dock are all in new colors. The landing pad and end caps are in silver (need to repaint).

thank you for all your hard work. The game is shaping up nicely! 😊

Dave/Servo

Sent from [Mail](https://go.microsoft.com/fwlink/?LinkId=550986) for Windows
